src/omnisharp/OmnisharpDownloader.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as utils from '../common';
+import * as fs from 'fs';
+
+export class OmnisharpDownloader {
+
+    public GetLatestExperimentVersion(): string {
```

Comment:
Since this tells you the most recently installed version, maybe `GetLatestInstalledExperimentalVersion`?